tests/integration/fuzzing/parameterized_test.cairo

```diff
@@ -2,43 +2,79 @@
 
 from starkware.cairo.common.math import assert_le
 
+////////// no data - broken test
+
 @external
-func setup_examples() {
-    %{
-        max_examples(5)
+func setup_no_data() {
+    return ();
+}
+
+@external
+func test_no_data{syscall_ptr: felt*, range_check_ptr}(a: felt, b: felt) {
+    return ();
+}
 
+////////// only examples
+
+@external
+func setup_only_examples() {
+    %{
         example(a=3, b=4)
-        given(a = strategy.integers(10, 20), b = strategy.integers(30, 40))
         example(b=6, a=5)
     %}
     return ();
 }
 
 @external
-func test_examples{syscall_ptr: felt*, range_check_ptr}(a: felt, b: felt) {
+func test_only_examples{syscall_ptr: felt*, range_check_ptr}(a: felt, b: felt) {
     assert_le(0, a);
     assert_le(0, b);
     assert_le(a, b);
     return ();
 }
 
+////////// only given
+
 @external
-func setup_examples_without_given() {
+func setup_only_given() {
     %{
+        given(a = strategy.integers(10, 20), b = strategy.integers(30, 40))
+    %}
+    return ();
+}
+
+@external
+func test_only_given{syscall_ptr: felt*, range_check_ptr}(a: felt, b: felt) {
+    assert_le(0, a);
+    assert_le(0, b);
+    assert_le(a, b);
+    return ();
+}
+
+////////// given and examples
```

Comment:
```suggestion
```
It is redundant with function name

Reply:
[modify cairo test style](https://github.com/software-mansion/protostar/pull/995/commits/fe79f3f1bc58c4a01034e9b71e3b04621944ba90)